### ISSUE TITLE
implement filename attribute: director's view

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#1622]
I just want to show it together with the description. It does not really make all that much sense here but I'll do it for closure's sake.

**Comment by urapadmin** [comment]
not trivial: Attributes of the image table itself are not part of the cql result. Only the uid is. That is also the reason for #1630.
It would need a lookup operation that fetches those attributes from the image table without including it in the context. Needs more thinking.

**Comment by urapadmin** [comment]
postponed to after the field seasons

**Comment by urapadmin** [comment]
I can spend my time with more important things

**Comment by urapadmin** [comment]
while this is true, it was really just a minute after #1599 was solved. 
### kiosk 1.2.10
So if there is a filename for a file in the edit file dialog, it will be displayed here.
